tests/cloner_test.go

```diff
@@ -138,6 +138,33 @@ var _ = Describe("all clone tests", func() {
 			Expect(uploader.DeletionTimestamp).To(BeNil())
 		})
 
+		It("should recreate and reclone target pvc if it was deleted", func() {
```

Comment:
So this test will actually run Serially since it's under `Describe("[rfe_id:1277][crit:high][vendor:cnv-qe@redhat.com][level:component]Cloner Test Suite", Serial, func()`
Can we maybe move it around so it can run parallel, or is there a constraint I am missing?

Reply:
yes sure thanks for noticing

Reply:
I actually dont see there is the serial decorator there, are you sure? @akalenyu